kumascript/macros/specs.ejs

```diff
@@ -0,0 +1,20 @@
+<%
+/*
+Placeholder to render a specification section with spec_urls from BCD
+
+Parameters
+
+$0 – A query string indicating for which feature to retrieve specification URLs for.
+
+Example calls
+
+{{specs}}
+{{specs("html.element.abbr")}}
+
+*/
+
+var query = $0 || env['browser-compat'];
```

Comment:
Smart!

But do you mind adding 
```
if (!query) {
  throw new Error("No first query argument or 'browser-compat' front-matter value passed");
}
```
which is what we do in `Compat.ejs`. And if you can improve on the error message, feel free to do so.